source/deployment/sso-ldap.md

```diff
@@ -49,9 +49,35 @@ Notes:
 
 Organizations using multiple domains can integrate with Mattermost using a "Forest" configuration to bring together multiple domains. Please see [Forests as Collections of Domain Controllers that Trust Each Other](https://technet.microsoft.com/en-us/library/cc759073%28v=ws.10%29.aspx?f=255&MSPPError=-2147217396) for more information.
 
-### Troubleshooting
+### Troubleshooting / FAQ
 
-The following are troubleshooting suggestions on common error messages and issues.
+The following are frequently asked questions and troubleshooting suggestions on common error messages and issues.
+
+##### When I first set up and synchronize LDAP, are the users automatically created in Mattermost? 
+
+No, the users are created on first log in. 
```

Comment:
This isn't very clear, maybe "No, each user is created on their first login."

Reply:
Updated.

Reply:
Updated.